components/illustratedmessage/index.css

```
@@ -11,103 +11,145 @@ governing permissions and limitations under the License.
-  --spectrum-illustrated-message-description-font-weight: var(--spectrum-body-sans-serif-font-weight);
-  --spectrum-illustrated-message-description-font-style: var(--spectrum-body-sans-serif-font-style);
-  --spectrum-illustrated-message-description-font-size: var(--spectrum-illustrated-message-body-size);
-  --spectrum-illustrated-message-description-line-height: var(--spectrum-body-line-height);
-  --spectrum-illustrated-message-description-color: var(--spectrum-body-color);
-
-  /* CJK (Chinese, Japanese, and Korean) language support */
-  &:lang(ja),
-  &:lang(zh),
-  &:lang(ko) {
-    --spectrum-illustrated-message-title-font-size: var(--spectrum-illustrated-message-cjk-title-size);
-  }
+	/* Size & Spacing */
+	--spectrum-illustrated-message-description-max-inline-size: var(--spectrum-illustrated-message-maximum-width);
+	--spectrum-illustrated-message-heading-max-inline-size: var(--spectrum-illustrated-message-maximum-width);
+	--spectrum-illustrated-message-title-to-heading: var(--spectrum-spacing-400);
+	--spectrum-illustrated-message-heading-to-description: var(--spectrum-spacing-75);
+	--spectrum-illustrated-message-illustration-to-heading: var(--spectrum-spacing-200);
+	--spectrum-illustrated-message-illustration-to-content: var(--spectrum-spacing-200);
+	--spectrum-illustration-message-description-to-action: var(--spectrum-spacing-300);
+
+	/* Illustration */
+	--spectrum-illustrated-message-illustration-color: var(--spectrum-neutral-visual-color);
+	--spectrum-illustrated-message-illustration-accent-color: var(--spectrum-accent-visual-color);
+	--spectrum-illustrated-message-illustration-size: 96px;
+
+	/* Title */
+	--spectrum-illustrated-message-title-font-family: var(--spectrum-sans-font-family-stack);
+	--spectrum-illustrated-message-title-font-weight: var(--spectrum-heading-sans-serif-font-weight);
+	--spectrum-illustrated-message-title-font-style: var(--spectrum-heading-sans-serif-font-style);
+	--spectrum-illustrated-message-title-font-size: var(--spectrum-title-size-xl);
+	--spectrum-illustrated-message-title-line-height: var(--spectrum-heading-line-height);
+	--spectrum-illustrated-message-title-color: var(--spectrum-heading-color);
+
+	/* Description */
+	--spectrum-illustrated-message-description-font-family: var(--spectrum-sans-font-family-stack);
+	--spectrum-illustrated-message-description-font-weight: var(--spectrum-body-sans-serif-font-weight);
+	--spectrum-illustrated-message-description-font-style: var(--spectrum-body-sans-serif-font-style);
+	--spectrum-illustrated-message-description-font-size: var(--spectrum-body-size-s);
+	--spectrum-illustrated-message-description-line-height: var(--spectrum-body-line-height);
+	--spectrum-illustrated-message-description-color: var(--spectrum-body-color);
+
+	/* CJK (Chinese, Japanese, and Korean) language support */
+	&:lang(ja),
+	&:lang(zh),
+	&:lang(ko) {
+		--spectrum-illustrated-message-title-font-size: var(--spectrum-illustrated-message-cjk-title-size);
```

We'll change this one to the component-specific token too (double-check Figma to make sure this is what the token is called), then update it again in small and large for those tokens:

```suggestion
		--spectrum-illustrated-message-title-font-size: var(--spectrum-illustrated-message-medium-cjk-title-font-size);
```